src/azure-map/interfaces/am-config.ts

```diff
@@ -0,0 +1,26 @@
+
+declare type AmConfig = ServiceOptions & CameraOptions & StyleOptions & UserInteractionOptions;
+  // serviceOptions: ServiceOptions;
```

Comment:
I think those comments are unnceccessary. Can we remove them?

Reply:
Sure

Reply:
I think you fogot this :)